src/msbuild/DNNE.targets

```diff
@@ -124,12 +124,14 @@ DNNE.targets
       they flow through project references.
   -->
   <ItemGroup Condition="'$(DnneBuildExports)' == 'true' AND '$(DnneAddGeneratedBinaryToProject)' == 'true'">
-    <None Include="@(DnneNativeExportsInput->'$(DnneNativeExportsBinaryPath)%(OutputFileName)')">
+    <Content Include="@(DnneNativeExportsInput->'$(DnneNativeExportsBinaryPath)%(OutputFileName)')">
       <Link>%(Filename)%(Extension)</Link>
+      <TargetPath>%(Filename)%(Extension)</TargetPath>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
       <Visible>false</Visible>
-    </None>
-    <None Include="$(DnneGeneratedBinPath)/$(DnneNativeExportsBinaryName).pdb" Condition="$([MSBuild]::IsOsPlatform('Windows'))">
+      <PublishState>Included</PublishState>
+    </Content>
+    <None Include="$(DnneGeneratedBinPath)/$(DnneNativeExportsBinaryName).pdb" Condition="$([MSBuild]::IsOsPlatform('Windows')) AND '$(DnneBuildExports)' == 'true'">
```

Comment:
This condition is on the enclosing `ItemGroup`, it is also needed here?

Reply:
Not needed here, I just missed the condition on the ItemGroup.

What exactly are the semantics of DnneBuildNativeExports vs DnneGenerateExports vs DnneAddGeneratedBinaryToProject? I'd imagine the .c and .h files still be copied if DnneBuildNativeExports != true but DnneGenerateExport == true, but with the condition on the ItemGroup it looks like they won't be copied to the output folder.

Reply:
`DnneGenerateExports` - Toggles if the `.h` and `.c` files should be generated based on the .NET assembly. It fails if there aren't any exports.
`DnneBuildNativeExports` - Toggles if the native binary with the exports are compiled.
`DnneAddGeneratedBinaryToProject` - Toggles if the native binary should flow with the project references.